app/ts/components/Navbar.tsx

```diff
@@ -20,7 +21,7 @@ export const Navbar = ({
 							<p className='font-bold'>{provider.value.walletAddress}</p>
 							<span className='text-gray-400 text-sm'>{provider.value.chainId === 1n ? 'Mainnet' : `Network ${provider.value.chainId.toString()}`}</span>
 						</div >
-						<div class='h-12 w-12 bg-white/20'></div>
+						<Blockie seed={provider.value.walletAddress} size={12} />
```

Comment:
It's important here that this is lowercased string address and not check summed. Check summed produces wrong blockie. I guess this one is checksummed, so add .tolower() here I think?